css/includes/_fonts.scss

```diff
@@ -29,11 +29,21 @@
  * ---------------------------------------------------------------------
  */
 
-@font-face {
-    font-family: Inter;
-    src: url("../fonts/Inter-Variable/Inter-VariableFont.ttf");
-}
+@import "~@fontsource/inter/scss/mixins";
 
+$fontDir: "../css/lib/fontsource/inter/files";
+
+@include fontFace($weight: 100);
```

Comment:
just a note to say their mixin name is not the best...

Reply:
I agree. I thought about isolating this in a dedicated font css, but it would have require to ensure that everyone is loading both palette and font CSS, and it seemed to late to me to introduce such a change.